launch_pytest/launch_pytest/plugin.py

```diff
@@ -316,7 +316,12 @@ def pytest_pyfunc_call(pyfuncitem):
         yield
         return
 
-    func = pyfuncitem.obj
+    # Store the original unwrapped function to avoid re-wrapping on reruns.
+    # This prevents issues with pytest plugins like pytest-flaky or pytest-rerunfailures,
+    # which reuse the same pyfuncitem across multiple test runs.
+    func = getattr(pyfuncitem, "_launch_pytest_original_obj", pyfuncitem.obj)
+    pyfuncitem._launch_pytest_original_obj = func
+    
```

Comment:
The diff view here doesn't easily show it, but this line contains whitespace:

```
  ./launch_pytest/plugin.py:324:1: W293 blank line contains whitespace
```

https://build.ros2.org/job/Rpr__launch__ubuntu_noble_amd64/139/testReport/junit/launch_pytest.test/test_flake8/test_flake8/